### PR TITLE
feat(styles, documentation): textarea component

### PR DESCRIPTION
**Pull request by leagrdv**

### CODE REVIEW

File: packages/documentation/src/stories/components/textarea/textarea.stories.ts [hunk 1]
```diff
@@ -204,7 +160,7 @@ function renderTextarea(args: Args, context: StoryContext) {
       id=${context.id}
       class=${classes}
       defaultValue=${args.value ?? nothing}
-      placeholder=${useAriaLabel ? args.label : ''}
+      placeholder=${useAriaLabel ? args.label : 'My placeholder'}
```
Comment (alizedebray) [inline packages/documentation/src/stories/components/textarea/textarea.stories.ts:163]: I know this was already there before but using labels as placeholders is a bad practice so I would remove this logic: 
```suggestion
      placeholder="My placeholder"
```

File: packages/documentation/src/stories/components/textarea/textarea.stories.ts [hunk 2]
```diff
@@ -227,31 +183,11 @@ export const Default: Story = {};
 export const FloatingLabel: Story = {
   parameters: {
     controls: {
-      exclude: ['Hidden Label', 'Size', 'Rows', 'Helper Text', 'Disabled', 'Validation'],
+      exclude: ['Hidden Label', 'Rows', 'Helper Text', 'Disabled', 'Validation'],
     },
   },
```
Comment (alizedebray) [inline packages/documentation/src/stories/components/textarea/textarea.stories.ts:188]: These excluded controls are legacy code, when you see them you can remove them completely.

File: packages/styles/src/components/form-textarea.scss [hunk 3]
```diff
@@ -1,17 +1,188 @@
...
 
 textarea.form-control {
-  &::-webkit-resizer {
-    background-image: url("data:image/svg+xml,%3Csvg viewBox='0 0 15 15' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath fill='#{forms.$input-border-color}' d='M0 12H3V15H0V12Z'/%3E%3Cpath fill='#{forms.$input-border-color}' d='M6 12H9V15H6V12Z'/%3E%3Cpath fill='#{forms.$input-border-color}' d='M6 6H9V9H6V6Z'/%3E%3Cpath fill='#{forms.$input-border-color}' d='M12 6H15V9H12V6Z'/%3E%3Cpath fill='#{forms.$input-border-color}' d='M12 0H15V3H12V0Z'/%3E%3Cpath fill='#{forms.$input-border-color}' d='M12 12H15V15H12V12Z'/%3E%3C/svg%3E");
-    background-repeat: no-repeat;
-    background-size: 75%;
-  }
+  display: block;
+  width: 100%;
+  font-family: tokens.get('body-font-family', elements.$post-body);
```
Comment (alizedebray) [inline packages/styles/src/components/form-textarea.scss:16]: Isn't it inherited?

File: packages/styles/src/components/form-textarea.scss [hunk 4]
```diff
@@ -1,17 +1,188 @@
...
+  @include forms-mx.textarea-resizer-svg(#050400);
 
   &[disabled] {
-    &::-webkit-resizer {
-      background-image: url("data:image/svg+xml,%3Csvg viewBox='0 0 15 15' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath fill='#{forms.$input-disabled-border-color}' d='M0 12H3V15H0V12Z'/%3E%3Cpath fill='#{forms.$input-disabled-border-color}' d='M6 12H9V15H6V12Z'/%3E%3Cpath fill='#{forms.$input-disabled-border-color}' d='M6 6H9V9H6V6Z'/%3E%3Cpath fill='#{forms.$input-disabled-border-color}' d='M12 6H15V9H12V6Z'/%3E%3Cpath fill='#{forms.$input-disabled-border-color}' d='M12 0H15V3H12V0Z'/%3E%3Cpath fill='#{forms.$input-disabled-border-color}' d='M12 12H15V15H12V12Z'/%3E%3C/svg%3E");
+    @include forms-mx.textarea-resizer-svg(#696864);
+  }
+
+  &:focus {
+    @include utilities.focus-style;
```
Comment (alizedebray) [inline packages/styles/src/components/form-textarea.scss:37]: The `focus-style` mixin already includes the focus selectors.

**Review by alizedebray — APPROVED** [review]

File: packages/styles/src/components/form-textarea.scss (hunk 3 above)
Comment (leagrdv) [reply packages/styles/src/components/form-textarea.scss]: the font-family is, I removed it but the other two aren't (display is inline-block and width is not 100% of its container)

**Pull request merged**